configfs.c

```diff
@@ -214,10 +216,54 @@ void tcmu_reset_netlink(void)
 	rc = tcmu_cfgfs_set_u32(path, 1);
 	if (rc) {
 		tcmu_warn("Could not reset netlink: %d\n", rc);
-		return;
+		return rc;
 	}
 
 	tcmu_dbg("reset netlink done\n");
+	return 0;
+}
+
+int tcmu_cfgfs_dev_block(struct tcmu_device *dev)
+{
+	int rc;
+
+	if (!tcmu_cfgfs_file_is_supported(dev, "action")) {
```

Comment:
@mikechristie can you consider renaming tcmu_cfgfs_file_is_supported() to something better :-)

Reply:
tcmu_cfgfs_dev_file_is_supported is also okay.

For better readability, we should probably write a wrapper function tcmu_dev_is_ringreset_supported(), which should call the generic configfs file/dir existence checker tcmu_cfgfs_dev_file_is_supported()

Reply:
Ok sounds good.

Reply:
I did tcmu_cfgfs_dev_file_is_supported. I don't want to end up creating a wrapper for every new file we add.

Reply:
I ended up just killing the function. The code using it is a little odd, so I do not think it was needed.